onnxruntime/python/tools/transformers/fusion_qordered_layernorm.py

```diff
@@ -0,0 +1,95 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from logging import getLogger
```

Comment:
I would add a brief docstring to summarize the module:

https://google.github.io/styleguide/pyguide.html#382-modules